server/controllers/finance/reports/cashflow/index.js

```diff
@@ -189,6 +192,9 @@ function processingWeekCashflow(params) {
     throw new BadRequest('Cashbox is missing', 'ERRORS.BAD_REQUEST');
   }
 
+  params.dateFrom = moment(params.dateFrom).format('YYYY-MM-DD').toString();
```

Comment:
Why is this so complex?  All other date ranges seem to work with the simply JS and SQL parsing...